### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by breyell**
Made this to update packages for Google Analytics but did a couple other things, too.
* Fixed linter errors
* Added dist folder to `.gitignore`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 *.DS_Store
 npm-debug.log
 node_modules
+dist
```
Comment (abea) [inline .gitignore:6]: The dist directory should be deleted once deployment is complete, so this shouldn't be needed.

File: package.json [hunk 2]
```diff
@@ -16,15 +16,15 @@
   "author": "Apostrophe Technologies, Inc.",
   "license": "MIT",
   "devDependencies": {
-    "@vuepress/plugin-google-analytics": "^1.5.4",
```
Comment (abea) [inline package.json:19]: Was there a bug in the earlier version of the GA plugin?

**Review by abea — APPROVED** [review]
I don't see any issues with this. I would suggest confirming that this does the analytics change you're looking for.

**Comment by breyell** [comment]
After looking into this it seems like there [isn't support for the new GA](https://github.com/vuejs/vuepress/issues/2713) yet. I'll leave merging or closing up to you.

**Pull request merged**